plugins/autostart/src/lib.rs

```diff
@@ -103,6 +137,7 @@ pub struct Builder {
     #[cfg(target_os = "macos")]
     macos_launcher: MacosLauncher,
     args: Vec<String>,
+    prefered_name: PreferedName,
```

Comment:
Since this is for the auto start entry name corresponding to `set_app_name`, maybe we could just call it `app_name`